pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -500,6 +500,16 @@ func (b *ConsoleServerCLIConfigBuilder) customization() Customization {
 			}
 		}
 
+		conf.Perspectives = perspectives
```

Comment:
Let's add a unit test that confirms that the dev perspective is not disabled when there is any perspective config.

Reply:
I wonder if this is a little odd since if I add a config to customize perspective `foo` it could also enable the `dev` perspective when it was disabled before because there was no config. We might want to change the logic here to only enable the `dev` perspective if the `admin` perspective is disabled (or disabled for at least some users due to an access review check).

What do you guys think?